functions.php

```diff
@@ -433,6 +433,20 @@ function twenty_twenty_one_scripts() {
 }
 add_action( 'wp_enqueue_scripts', 'twenty_twenty_one_scripts' );
 
+/**
+ * Enqueue block editor script.
+ *
+ * @since 1.0.0
+ *
+ * @return void
+ */
+function twentytwentyone_block_editor_script() {
+
+	wp_enqueue_script( 'twentytwentyone-unregister-block-style', get_theme_file_uri( '/assets/js/unregister-block-style.js' ), array( 'wp-blocks', 'wp-dom' ), wp_get_theme()->get( 'Version' ), true );
```

Comment:
It's a little annoying to have to enqueue JS just for this. But the core block styles are registered on the client side, so the [server-side registration helper](https://developer.wordpress.org/block-editor/developers/filters/block-filters/#unregister_block_style) does not work for this.

Reply:
Annoying indeed... But I can't think of a better solution either :shrug: 
I'm good with this. It's annoying, but it's all we can do.